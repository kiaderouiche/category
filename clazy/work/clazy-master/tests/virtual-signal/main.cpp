#include <QtCore/QObject>

class MyObj : public QObject
{
    Q_OBJECT
public:
    void foo();
signals:
    void mySig(); // OK
    virtual void myVirtualSig(); // Warn
public Q_SLOTS:
    void mySlot();
};

