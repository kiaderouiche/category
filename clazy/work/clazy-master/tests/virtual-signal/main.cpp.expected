virtual-signal/main.cpp:10:5: warning: signal is virtual [-Wclazy-virtual-signal]
